docs/extend/install-configure-zos-extensions.md

```diff
@@ -0,0 +1,144 @@
+# Install and Configure Zowe Server Component
+
+## Install with zowe-install-component.sh <Badge text="Technical Preview"/>
+
+_Note: this section is for technical preview and we are happy to hear any feedbacks. Content in this section may be changed or improved in the future._
+
+_Note: this feature is added with Zowe v1.19.0 release._
+
+From v1.19.0, Zowe ships a `bin/zowe-install-components.sh` tool to help you install any Zowe server component (extension). Zowe core components are also installed with this tool. To make the tool works better, the component to be installed is suggested to follow [Zowe server component package format standard](packaging-zos-extensions.md#zowe-server-component-package-format).
+
+The tool can be executed from z/OS USS, and it takes these command line parameters:
+
+- **`-o|--component-file`**: This is required. It's the path to the component package or directory.
+- **`-c|--component-name`**: This is optional. It's the name of the component. If `NODE_HOME` is defined, and the component has manifest file, this value is optional. Otherwise it's required.
+- **`-i|--instance-dir`**: This is optional. It's the path to Zowe instance directory. If this has a value, the script will also execute `bin/zowe-configure-component.sh` once it's installed.
+- **`-d|--target-dir`**: This is optional. It's the path to installation target directory. For native component, default value is `${ZOWE_ROOT_DIR}/components`. For non-native component, the script will check `ZWE_EXTENSION_DIR` variable value. If it's also empty, the script will fall back to default target directory, which is `/global/zowe/extensions`.
```

Comment:
Is the term 'native' component correct. I assume it means a core zowe component?